CodeCoverageQualityGate/CodeCoverageQualityService.ts

```diff
@@ -143,11 +143,13 @@ export class CodeCoverageQualityService {
                 const referenceCoberturaReportContent = await referenceCoberturaReport.contentsPromise;
                 coverageInfo = this.getModifiedCodeCoverage(referenceCoberturaReportContent, currentCoberturaReportContent);
             }
-        } catch (error: any) {
+        // TypeScript requires that catch variable must be 'any' or 'unknown'
```

Comment:
Can you change the block like something below instead of suppressing the rule?
@rwang-parasoft What do you think?
```
} catch (error) {
    if (error instanceof Error) {
        this.skipQualityGateWithWarning(`${error.message}`);
    } else {
        this.skipQualityGateWithWarning(`some message`);
    }
}
```